github/MainClass.py

```diff
@@ -789,6 +790,42 @@ def get_app(self, slug=github.GithubObject.NotSet):
             headers, data = self.__requester.requestJsonAndCheck("GET", f"/apps/{slug}")
             return GithubApp.GithubApp(self.__requester, headers, data, completed=True)
 
+    def get_installation_by_id(self, installation_id):
+        """
+        You must use a JWT to access this endpoint which can be generated using
+        :meth:`github.GithubIntegration.create_jwt`.
+
+        :calls: `GET /app/installations/:installation_id <https://docs.github.com/en/rest/reference/apps#get-an-installation-for-the-authenticated-app>`_
+        :param installation_id: int
+        :rtype: :class:`github.Installation.Installation`
+        """
+        assert isinstance(installation_id, int), installation_id
+        headers, data = self.__requester.requestJsonAndCheck(
+            "GET", f"/app/installations/{installation_id}"
+        )
+        return Installation.Installation(
+            self.__requester, headers, data, completed=True
+        )
+
+    def get_installation_by_repo(self, owner, repo):
```

Comment:
This is now implemented by `GithubIntegration.get_repo_installation(owner, repo)`.